scripts/tests/run_test_suite.py

```diff
@@ -18,21 +18,17 @@
 import logging
 import os
 import sys
-import tempfile
 import time
 import typing
 from dataclasses import dataclass, field
-from pathlib import Path
 
 import chiptest
 import click
 import coloredlogs
 from chiptest.accessories import AppsRegister
 from chiptest.glob_matcher import GlobMatcher
 from chiptest.test_definition import TestRunTime, TestTag
-from diskcache import Cache
-
-cache = Cache(os.path.join(tempfile.gettempdir(), 'yaml_runner_cache'))
+from yaml.paths_finder import PathsFinder
```

Comment:
Just as a side note, I think we really should rename this yaml directory in here in can collide with the actual yaml built in module name

Reply:
It makes sense.
I opened https://github.com/project-chip/connectedhomeip/issues/25375 as a followup.